### ISSUE TITLE
API_ID_PUBLISHED_FLOOD occurs during Fork app login after release-5.10 is released

### ISSUE THREAD

**Issue by shivanraptor** [TelegramMessenger/Telegram-iOS#117]
Our Fork users reported that they are unable to login in new installs of my Fork app.

The following error occurs: 

> Error 400: API_ID_PUBLISHED_FLOOD 

iOS version: 12.0.1 & 12.3.1
Fork version: release-5.9

I double checked the config. I'm using the API ID and hash registered in Telegram Core web portal (not using the original App ID in cloned release). 

This issue doesn't happen before release-5.10 is released. Maybe there are some policy changes? 

Anyone please help.

--- 

The error can be seen when a new user trying to confirm his phone number at the first login step. When the user taps Help, the error message will appear in a Compose Email box.

The API ID I used is **895xxx**, which is not the default API ID. Double checked.

**Comment by shivanraptor** [comment]
The same error also appears in `release-5.10`.

**Comment by laktyushin** [comment]
Please check that your own app id is correctly applied. You can set a breakpoint at this line — https://github.com/TelegramMessenger/Telegram-iOS/blob/7cf95a743cef5090f1f100414a6b2c894056c0db/submodules/TelegramCore/TelegramCore/Network.swift#L412 and check the value to be completely sure.
Also, are you sure you haven't mistakenly released a build with the provided demo app id in the past?

**Comment by shivanraptor** [comment]
Dear @laktyushin, 

Thanks for your prompt reply. I added the debug line after the Line 412 you quoted:

    print("API ID: \(apiId)")

and the result is still:

> API ID: 895xxx

Confirmed I'm using the same API ID I registered from Telegram Web Portal. Not sure if it's related, the App was quite popular, more than 1000+ downloads in a single day. Would the Telegram network think the API ID was abused due to high usage?
